pkg/controller/controller.go

```diff
@@ -522,3 +523,39 @@ func isInstanceReady(instance *v1alpha1.Instance) bool {
 
 	return false
 }
+
```

Comment:
@pmorie actually this PR is already merged into apimachinery. but we can't delete this code yet since it will be released only in 1.8. Might leave TODO there.